killbill-integration-tests/core/usage/test_capacity.rb

```diff
@@ -168,6 +178,9 @@ def test_multiple_units
 
       check_invoice_no_balance(second_invoice, 5.00, 'USD', '2015-03-01')
       check_usage_invoice_item(second_invoice.items[0], second_invoice.invoice_id, 5.00, 'USD', 'USAGE', 'basic-monthly', 'basic-monthly-evergreen', 'basic-monthly-usage1', '2015-02-01', '2015-03-01')
+      check_invoice_item_detail(second_invoice.items[0],
+                                [{:unit_type => 'bandwith-meg-sec', :unit_qty => 101, :unit_price => 5.0 },
+                                 {:unit_type => 'members', :unit_qty => 10, :unit_price => 5.0 }], 5.0)
```

Comment:
What should happen with `Detail Mode` ? Does this mode make sense have it for `Capacity` ?

Reply:
For `CAPACITY`, each tier has a set of prices (fixed and recurring) for any usage consumed up to certain limits, across multiple units (https://github.com/killbill/killbill/blob/master/catalog/src/test/resources/UsageExperimental.xml#L180-L189). So we will always have a single `USAGE` item for the complying tier level (regardless of the mode used).

In the aggregate mode, we can populate `item_details` with the JSON describing how much was used for each unit and which tier was matched (https://github.com/killbill/killbill-integration-tests/pull/13#discussion_r166222783).

In the detail mode, the `USAGE` item would be the same as what is is today. We cannot really populate the `quantity` field since it would span across multiple `units`. We could consider populating the JSON though (@sbrossie: thoughts?).